tests/js/realm-tests.js

```diff
@@ -22,6 +22,11 @@ var Realm = require('realm');
 var TestCase = require('./asserts');
 var schemas = require('./schemas');
 
+let pathSeparator = '/';
```

Comment:
There's `path.sep` in Node.js, but not in React Native. At least not to my knowledge :/